setup.py

```diff
@@ -121,7 +121,7 @@ def scmtools_parse(root,
 
     with open(cwd + '/requirements.txt') as fp:
 
-        INSTALL_REQUIRES = [L.strip() for L in fp if not L.startswith('git+')]
+        INSTALL_REQUIRES = [L.strip() for L in fp if (not L.startswith('git+') and not L.startswith('--extra-index-url'))]
```

Comment:
Is this a development-time hack that is not needed anymore?

Reply:
It has a niche case if you want to install from `test.pypi.org`.